lib/bamboo/plug/helper.ex

```diff
@@ -17,6 +17,12 @@ defmodule Bamboo.EmailPreviewPlug.Helper do
     |> Enum.join(", ")
   end
 
+  def format_values(values) when is_binary(values), do: values
+  def format_values(values) when is_list(values) do
+    Enum.join(values, ", ")
+  end
+  def format_values(values), do: inspect(values)
```

Comment:
I think for now `format_headers` is good and it can be renamed `format_values` later if it's needed in other parts of the library. We can leave the catch all just in case, but it would be good to add a test case of it